### PR TITLE
[FLINK-6816] [table] Fix wrong usage of Scala string interpolation in Table API

### PR DESCRIPTION
**Pull request by wuchong**
Thanks for contributing to Apache Flink. Before you open your pull request, please take the following check list into consideration.
If your changes take all of the items into account, feel free to open your pull request. For more information and/or questions please refer to the [How To Contribute guide](http://flink.apache.org/how-to-contribute.html).
In addition to going through the list, please provide a meaningful description of your changes.

- [x] General
  - The pull request references the related JIRA issue ("[FLINK-XXX] Jira title text")
  - The pull request addresses only one issue
  - Each commit in the PR has a meaningful commit message (including the JIRA id)

- [x] Documentation
  - Documentation has been added for new functionality
  - Old documentation affected by the pull request has been updated
  - JavaDoc for public methods has been added

- [x] Tests & Build
  - Functionality added by the pull request is covered by tests
  - `mvn clean verify` has been executed successfully locally or a Travis build has passed

### CODE REVIEW

File: flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/ProcTimeBoundedRangeOver.scala [hunk 1]
```diff
@@ -62,8 +62,8 @@ class ProcTimeBoundedRangeOver(
   private var function: GeneratedAggregations = _
 
   override def open(config: Configuration) {
-    LOG.debug(s"Compiling AggregateHelper: $genAggregations.name \n\n " +
-                s"Code:\n$genAggregations.code")
+    LOG.debug(s"Compiling AggregateHelper: ${genAggregations.name} \n\n " +
```
Comment (sunjincheng121) [inline flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/ProcTimeBoundedRangeOver.scala]: I think we supported remove the " ", i.e.: 
`${genAggregations.name} \n\n "` -> `${genAggregations.name} \n\n"`
If we do this change, the log print info as follows:
FROM 
```
Compiling AggregateHelper: MinMaxAggregateHelper 

   Code:
GeneratedAggregationsFunction(MinMaxAggregateHelper,
```
TO
```
Compiling AggregateHelper: MinMaxAggregateHelper 

Code:
GeneratedAggregationsFunction(MinMaxAggregateHelper,
```

**Comment by zentol** [comment]
+1.

**Comment by wuchong** [comment]
@sunjincheng121  I've addressed your suggestion.

**Comment by sunjincheng121** [comment]
Hi @wuchong Thanks for the update. 
+1 to merged.

**Comment by twalthr** [comment]
Thanks @wuchong. I will merge this.

**Pull request closed without merging**